src/style.css

```diff
@@ -1,3 +1,66 @@
+html {
+  --main-color: #00acdc;
+
+  font-family: Roboto, sans-serif, Arial;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  text-transform: uppercase;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
```

Comment:
The 'height' property in the .nav__link class may cause the links to appear larger than expected. Consider using 'line-height' or 'padding' for vertical alignment.